integration_tests/scheduled_cork_test.go

```diff
@@ -0,0 +1,166 @@
+package integration_tests
+
+import (
+	"context"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/peggyjv/sommelier/v3/x/cork/types"
+)
+
+func (s *IntegrationTestSuite) TestScheduledCork() {
+	s.Run("Bring up chain, and schedule a cork call to ethereum", func() {
+
+		// makes sure ethereum can be contacted and counter contract is working
+		count, err := s.getCurrentCount()
+		s.Require().NoError(err)
+		s.Require().Equal(int64(0), count.Int64())
+
+		s.T().Log("verify that there is one allowed addresses on the new chain")
```

Comment:
Removed that check. The better version of this would to be to just add the cellar to genesis state for this test, but genesis state is currently the same for all tests, that part of test definition needs to be improved